ncbimeta/NCBImetaUtilities.py

```diff
@@ -44,26 +44,49 @@ def table_exists(db_cur, table_name):
     query = "SELECT name FROM sqlite_master WHERE type='table' AND name='{}'".format(table_name)
     return db_cur.execute(query).fetchone() is not None
 
-def adv_xml_search(xml_root, xpath, column_name, xml_dict):
-    targ_xpath = ".//{}".format("/".join(xpath))
+def adv_xml_search(xml_root, targ_xpath, column_name, xml_dict):
```

Comment:
And if you find that your values reside within parent nodes (example below) feel free to remove/change `if result.getchildren() == 0`.

Example:
```
<foo>
    <bar>
        <key>Hello</key>
        <value>World</value>
        Hello, I'm a value!
    </bar>
    <bar>
        <key>Hello</key>
        <value>dlroW</value>
    </bar>
</foo>

Reply:
This is excellent! Much better to have explicit checking of the expected objects, rather than my wild try/excepts :) I've added this in commit c04e981, with one change:
```
if len(result.getchildren()) == 0:
```
I found that the ```getchildren()``` function returned a list object, and so I used ```len()``` to test for no children/empty list. So far your function works perfectly for all tests and example data I I've come across.

Reply:
Nice catch! Glad it's working correctly otherwise! 👍